lib/rex/post/meterpreter/ui/console/command_dispatcher/extapi/clipboard.rb

```diff
@@ -14,10 +14,15 @@ module Ui
 class Console::CommandDispatcher::Extapi::Clipboard
 
   Klass = Console::CommandDispatcher::Extapi::Clipboard
-
+  
+  STEP_SKIPPED_WOULD_OVERWRITE = 'Overwrite'
+  STEP_COMPLETED = 'Completed'
+  STEP_SKIPPED = 'Skipped'
+  STEP_COMPLETED_OVERWRITTEN = 'Overwritten'
+  
```

Comment:
I think we want these constants moved to here:

https://github.com/rapid7/metasploit-framework/blob/0f4c73b9789166d0983db7a3816893adbda3d044/lib/rex/post/meterpreter/extensions/stdapi/fs/file.rb#L29

That would allow them to be used in both places where they're needed. It makes sense to put them there and have `ui/console/command_dispatcher/extapi/clipboard.rb` reference `extensions/stdapi/fs/file.rb` because `file.rb` is already the dependency. You should be able to reference the constants from this file by using `Meterpreter::Extensions::Stdapi::Fs::File::STEP_...`. The name will be long but the constants won't be duplicated.